tests/Unit/Evolution/DiscontinuousGalerkin/Messages/Test_BoundaryMessage.cpp

```diff
@@ -73,9 +73,11 @@ void test_boundary_message(const gsl::not_null<Generator*> generator,
       neighbor_direction, element_id, volume_mesh, interface_mesh,
       subcell_size != 0 ? subcell_data.data() : nullptr,
       dg_size != 0 ? dg_data.data() : nullptr);
+  // Since we expect the copied message to have sent_across_nodes = true because
+  // that's set in the pack() function, we do `not sent_across_nodes` here
```

Comment:
If this is true independent of the value of `sent_across_nodes`, just make it a literal `true`.